src/components/AccessForm.vue

```diff
@@ -30,17 +30,22 @@
       ></el-input-number>
     </el-form-item>
     <el-form-item style="margin-top: 15px">
-      <el-button type="primary" @click="onSubmit('accessForm')"
-        >Create</el-button
+      <el-button
+        type="primary"
+        :loading="loading"
+        @click="onSubmit('accessForm')"
       >
+        Create
```

Comment:
"Request" probably makes more sense, since this is requesting a new execution token?

```suggestion
        Request
```